roles/xdebug/defaults/main.yml

```diff
@@ -0,0 +1,41 @@
+# XDebug Remote Debugging
+xdebug_remote_enable: 'false'
+xdebug_remote_connect_back: 'false'
+xdebug_remote_host: localhost
+xdebug_remote_port: '9000'
+xdebug_remote_log: /tmp/xdebug.log
+xdebug_idekey: XDEBUG
+xdebug_extended_info: 1
+xdebug_max_nesting_level: 200
+
+# XDebug Display Settings
+xdebug_var_display_max_children: 128
+xdebug_var_display_max_data: 512
+xdebug_var_display_max_depth: 3
+xdebug_force_display_errors: 0
+xdebug_force_error_reporting: 0
+xdebug_scream: 0
```

Comment:
nitpick: It might be nice if the `Display Settings` above were in the same order as they appear in the `xdebug.ini.j2` template.